doc/nrf/libraries/others/pcm_mix.rst

```diff
@@ -0,0 +1,29 @@
+.. _lib_pcm_mix:
+
+PCM (Pulse Code Modulation) audio mixer
+################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+The PCM mixer enables users to mix a PCM stream into another PCM stream.
+There is support for mixing a stereo stream into another stereo stream,
+or, combinations of mono to mono, or, mono to stereo channel left or right.
+
+Configuration
+*************
+
+To enable the library, set the :kconfig:option:`CONFIG_PCM_MIX` Kconfig option to ``y`` in the project configuration file :file:`prj.conf`.
```

Comment:
Need to confirm is this is the preferred method.

Reply:
What do you mean by this @koffes ?

Reply:
Have a look now.

Reply:
I meant the sentence "Need to confirm is this is the preferred method.", but looks like this is good now :)